src/arrayMethodSort.js

```diff
@@ -4,9 +4,39 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = compareStrings) {
+    if (this.length === 1) {
+      return this;
+    }
+
+    let count;
+
+    do {
+      count = 0;
+
+      for (let i = 1; i < this.length; i++) {
+        if (compareFunction(this[i - 1], this[i]) > 0) {
+          [[this[i - 1]], [this[i]]] = [[this[i]], [this[i - 1]]];
```

Comment:
I have no idea how the tests have been passed😅 But I suggest to make in more readable and understandable way
```
const previous = this[i - 1];
const current = this[i];

this[i - 1] = current;
this[i] = previous;
```

Reply:
[a, b] = [b, a] is the destructuring assignment that swaps the variables a and b.

i chose this method because it short and simple